### DISCUSSION TITLE
Create a raster (CHM or DTM) that matches an existing grid

### DISCUSSION THREAD

**Discussion by DeltaForest-DP** [r-lidar/lidR#639] [Q&A]
Hi, I'm trying to generate a CHM and DTM that match an existing product that sits on a grid that is offset from mine by a set of known coordinates. The command **grid_metrics(..., start=c(0.42, -0.2)**, ...) does this nicely for the point-cloud stats - is there a similar way to shift the raster in **rasterize_canopy()** and **rasterize_terrain()**?  (note that the projection is the same; just the grid used previously is shifted in x and y)

Thanks!

**Comment by Jean-Romain** [comment]
Provide your raster as a template

> **res**     numeric. The size of a grid cell in point cloud coordinates units. Can also be `RasterLayer` or a `stars` or a `SpatRaster` used as layout.
